suggestions for using sub domains, other typo fixes, table for registry

ms-s opened the pull request.

alandekok commented:
I've pulled over most of the changes manually, and done some touch-ups.  There are still a few things to address.

ms-s commented:
Thanks. I went through the changes in the latest [commit](https://github.com/FreeRADIUS/eap-arpa/commit/b117f4653d824951a7108110136b265a01bab5ad) and they mostly look good to me. I think we will need some discussion on the mailing list about the vendor sub domains. 

If you are not fundamentally opposed to the idea, I'd like to join as a co-author. I promise to help with editing and addressing comments on the list :), I don't need access to the [FreeRADIUS project space](https://github.com/FreeRADIUS/). I think once we move the draft to the [EMU working group space](https://github.com/emu-wg/) I could add myself as an author and help with editing the document.
